tests/sqla_models_tests.py

```diff
@@ -79,16 +79,16 @@ def test_db_column_types(self):
             # string
             "CHAR": GenericDataType.STRING,
             "VARCHAR": GenericDataType.STRING,
-            "NVARCHAR": GenericDataType.STRING,
-            "STRING": GenericDataType.STRING,
+            # "NVARCHAR": GenericDataType.STRING,   # MSSQL types; commeented out for now and will address in another PR
+            # "STRING": GenericDataType.STRING,
```

Comment:
Were these causing problems in tests? This test might need some refactoring, as it will potentially give different results on different engines. We could potentially simplify this a bit by only checking types that are common for all databases supported by CI, and later potentially adding a few db specific tests.